skypilot-dev/skypilot-dev.github.io
# Upgrade dependencies & CI workflows

This branch

- Updates all dependencies to their latest versions
- Adopts the latest CI workflows bundled with Toolchain

### To test

Run the site locally: `yarn run dev`, then verify that

- The site displays normally
- The scroll-to-location functionality works as expected
- The "Contact us" form still works
